components/components-en.html

```diff
@@ -25,7 +25,20 @@ <h2 id="download">Download link</h2>
 	<p>Download links are used for referencing and linking to non-HTML files on Canada.ca web pages.</p>
 	<div class="row">
 		<div class="col-sm-4">
-			{{>comp-download}}
+			<a class="gc-dwnld-lnk" href="#">
```

Comment:
We might need to define it as a independent component : `gc-dwnld-lnk` ; `gc-dwnld`; `gc-dwnld-txt`; `gc-dwnld-info`